cmd/ddev/cmd/global_dotddev_assets/commands/host/heidisql

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+## #ddev-generated: If you want to edit and own this file, remove this line.
+## Description: Run HeidiSQL against current db
+## Usage: heidisql
+## Example: "ddev heidisql"
+## OSTypes: linux,windows
```

Comment:
The other script commands show only if the executable is installed. Seems we ought to be able to do that here as well, except for the problem of two-environments one-machine.